test/tests.py

```diff
@@ -65,8 +65,8 @@ def active_test(fullname):
 bat_sfx = batch_suffix()
 exe_sfx = '.exe' if is_windows() else ""
 
-mini_python = [join(simics_base_path(), host_type(), "bin",
-                    "mini-python" + exe_sfx)]
+mini_python = [join(testparams.project_path(), "bin",
+                    f"mini-python{batch_suffix()}")]
```

Comment:
I missed the `{batch_suffix()}` part. I'll run a quick Test W just to be sure it works OK there.

Reply:
... or so I say yet TC decided to rebuild Build W for no reason. bah.

Reply:
[Here's the run](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=12947896&buildTypeId=SimicsCiSimics_MonoTests_TestWFast). All failures are unrelated, and the t126 suite is no slower, so I'd deem this good.